openassessment/xblock/xml.py

```diff
@@ -760,7 +761,7 @@ def serialize_examples_to_xml_str(assessment):
         A unicode string of the XML serialized examples.
 
     """
-    examples = assessment.get('examples', [])
+    examples = update_assessments_format([assessment])[0].get('examples', [])
```

Comment:
`update_assessments_format` takes a list of assessments so we put the `assessment` in a list, and after the list has been updated we retrieve the `assessment` with `[0]`.